_posts/2017-03-02-security-review.md

```diff
@@ -17,7 +17,7 @@ As would be expected of any good security review, DevCore did find several issue
 
 #### 1. Insufficient e-mail address validation
 
-When logging in with an e-mail address, a user could enter an address like "foo@evil.com,bar@example.com". In this case, login e-mails would be sent to _both_ addresses, while Sandstorm would treat the user as if they were a member of the latter domain, "example.com". Hence, an attacker could specify their real address as the first address and a fake address at a domain of their choice as the second, in order to appear to be a member of the domain. Servers that use Sandstorm's organizational features (until recently only available as part of the [now-defunct](https://sandstorm.io/news/2017-02-06-sandstorm-returning-to-community-roots) product "Sandstorm for Work") can be configured to automatically promote members of a domain to full users, allowing them to install apps, create grains, and discover the identities of other members of the organization. Such users would **not** receive access to any other grains on the server, but an attacker could install their own apps that consume the server's resources, including CPU, RAM, and disk. An attacker could also combine this vulnerability with one of the others below, all of which require full user status as a starting point.
```

Comment:
A bit troubling that Github appears to be failing to highlight the change on this line.